source/install/requirements.rst

```diff
@@ -55,8 +55,8 @@ Mobile Apps
 .. csv-table::
     :header: "Operating System", "Technical Requirement"
 
-    "iOS", "iPhone 5s devices and later with iOS 11+"
-    "Android", "Android devices with Android 7+"
+    "iOS", "iPhone 5s devices and later with iOS 12+"
+    "Android", "Android devices with Android 8+"
```

Comment:
We'll need to make sure we RETAIN the old builds for Android 7 users in the play store (they just can't access any future updates)

Reply:
I'd say drop iOS 12 specially as iOS15 is being release in a couple of months. And Apple is dropping support for iOS 12.

As for Android 8, what is the reason behind this?

Reply:
@esethna on the other hand, is it really a mistake? What about those ppl on Android 5 using 1.14 still? Do we have numbers on this? Do we have numbers on how many of them are using Android 7. After that is answered we could look into being a mistake or not

Reply:
Bases on the [guidelines](https://handbook.mattermost.com/operations/research-and-development/product/development-process/software-requirements#mattermost-software-requirements) (which is in the PR description), we support latest and next-to-latest versions by iOS and supported versions by Google.

Reply:
Alright based in this we can support Android 8, but that does not mean we stop to distribute to Android 7 unless we in fact are willing to say goodbye to those using Android 7.

In case of iOS we should support iOS 13+ as in a month time 15 gets out to the public.

We need to look at some numbers. Fact is that I'm 100% in agreement of having people use devices that run the latest software, but we should also look at what is real and what not. Normally iOS users tend to update/upgrade way faster than android

Reply:
@amyblais are we changing this to iOS13 as per Elias's suggestion, or waiting for iOS 15 to come out first?

Reply:
I'm asking Support team for additional thoughts on this PR.

Reply:
I've kept Android at 7+ based on support team's feedback, and I've kept iOS at 12+ for now.